\\\\
\\ src/prog/gpm-root.y:449: error: 'f_debug_one' defined but not used
diff -rdup gpm-1.99.5.old6/src/prog/gpm-root.y gpm-1.99.5/src/prog/gpm-root.y
--- gpm-1.99.5.old6/src/prog/gpm-root.y	2008-04-01 23:43:48.000000000 +0200
+++ gpm-1.99.5/src/prog/gpm-root.y	2008-04-15 13:40:36.000000000 +0200
@@ -445,6 +445,7 @@ void f__fix(struct passwd *pass)
 }
 
 /*---------------------------------------------------------------------*/
+#ifdef DEBUG
 static int f_debug_one(FILE *f, Draw *draw)
 {
    DrawItem *ip;
@@ -467,6 +468,7 @@ static int f_debug_one(FILE *f, Draw *dr
 #undef LINE
    return 0;
 }
+#endif
 
 int f_debug(int mode, DrawItem *self, int uid)
 {
